web/src/routes/LoginPage.tsx

```diff
@@ -1,5 +1,127 @@
 import React from 'react';
 
-export default function LoginPage() {
-  return <div>LoginPage</div>;
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import BackgroundImage from '../assets/main-page.webp';
+import Header from '../components/Header';
+import { useTheme } from '../hooks/useTheme';
+
+// 로그인 페이지 모르는 점
+// 1. 가로 짝대기 어떻게 생성하는지
+// 2. 로그인 제목이 더 굵게 안됨.
+// 3. 이메일과 비밀번호 font-color가 바뀌지 않음.
+// 4. 로그인컨테이너의 최소폭을 설정하고 싶은데 모름.
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url(${BackgroundImage}) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+`;
+
+const LoginContainer = styled.div`
+  padding: 9vw;
+  margin-left: 20vw;
+  margin-right: 20vw;
+  min-width : 20vh;
+  height: 50vh;
+  background-color: ${(props) => props.theme.colors.container};
+  box-shadow: 0 10px 15px rgba(0, 0, 0, 0.3);
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+
```

Comment:
Please remove all whitespaces and blank spaces. Or please run

```bash
$ npm run lint:fix && npm run format
```

FYI. In your local Visual Studio Code, install extensions for **Prettier**, **ESLint** and check `Format on Save` to apply auto save and format.

Reply:
Install extensions Prettier and Eslint, and npm run lint:fix. but it was error [ No files matching the pattern "'./src/**/*.{ts,tsx,js,jsx}'" were found. Please check for typing mistakes in the pattern. ]

Reply:
Well let me figure it out